test/functional/services/test_subjects.js

```diff
@@ -33,12 +35,12 @@ export function TestSubjectsProvider({ getService }) {
   const defaultFindTimeout = config.get('timeouts.find');
 
   class TestSubjects {
-    async exists(selector, timeout = 1000) {
+    async exists(selector, timeout = WAIT_FOR_EXISTS_TIME) {
       log.debug(`TestSubjects.exists(${selector})`);
       return await find.existsByDisplayedByCssSelector(testSubjSelector(selector), timeout);
     }
```

Comment:
@Stacey-Gammon I think it would be simpler to just remove the timeouts from the calls in test_subjects because they all have defaults in the find functions.

Reply:
Yea, though then every call to exists, when something legit doesn't exist, will take 10seconds.  Maybe we are okay with that though?  Rather take longer and be less flaky? and hope people switch to use `existsOrFail` instead, and less checks for missing things (which is slow).

Reply:
Ah, right.  Keep it as-is.